addon/components/custom-subsidy-form-fields/climate-subsidy-costs-table/table-row-custom-action.js

```diff
@@ -0,0 +1,416 @@
+import Component from '@glimmer/component';
+import rdflib from 'browser-rdflib';
+import { v4 as uuidv4 } from 'uuid';
+import { tracked } from '@glimmer/tracking';
+import { A } from '@ember/array';
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import { scheduleOnce } from '@ember/runloop';
+
+import {
+  RDF,
+  XSD,
+  removeSimpleFormValue,
+} from '@lblod/submission-form-helpers';
+
+const MU = new rdflib.Namespace('http://mu.semte.ch/vocabularies/core/');
+const QB = new rdflib.Namespace('http://purl.org/linked-data/cube#');
+
+const climateBaseUri = 'http://data.lblod.info/vocabularies/subsidie/climate/';
+const climateTableBaseUri = 'http://data.lblod.info/climate-tables';
+
+const tableEntryBaseUri = 'http://data.lblod.info/id/climate-table/row-entry';
+const ClimateEntryType = new rdflib.NamedNode(`${climateBaseUri}ClimateEntry`);
+const climateEntryPredicate = new rdflib.NamedNode(
+  `${climateBaseUri}climateEntry`
+);
+const climateEntryCustomAction = new rdflib.NamedNode(
+  `${climateBaseUri}customAction`
```

Comment:
This is added to the custom rows so that we can retrieve the data in the parent component on load. Let me know if there are better ways of doing this or if I should use a different name.